drivers/wifi/esp/esp.c

```diff
@@ -698,16 +698,16 @@ static void esp_init_work(struct k_work *work)
 		SETUP_CMD_NOHANDLE("AT"),
 		/* turn off echo */
 		SETUP_CMD_NOHANDLE("ATE0"),
-#if defined(CONFIG_WIFI_ESP_AT_VERSION_2_0)
-		SETUP_CMD_NOHANDLE("AT+CWAUTOCONN=0"),
-#endif
 		SETUP_CMD_NOHANDLE("AT+UART_CUR="_UART_CUR),
 		/* enable multiple socket support */
```

Comment:
Looks like this comment should be moved with the AT+CIPMUX=1 command.

Reply:
That is correct and now fixed. Thanks